File: orders_plot.R
library(ggplot2)

orders <- read.csv("data/orders.csv")

print("First rows")
print(head(orders))

print("Rows and columns")
print(dim(orders))

ggplot(orders, aes(x = quantity, y = order_total, color = category)) +
  geom_point()

File: orders_plot.py
import pandas as pd
import seaborn as sns
import matplotlib.pyplot as plt

orders = pd.read_csv("data/orders.csv")

print("First rows")
print(orders.head())

print("Rows and columns")
print(orders.shape)

sns.scatterplot(x="quantity", y="order_total", hue="category", data=orders)
plt.show()
